fix(#18): Fix rounding errors, which can cause false negatives

carmanchris31 opened the pull request:
Fixes #18

krzkaczor commented:
Thanks, for this PR! I am wondering if the better solution wouldn't be just to round down numbers during formatting? `26/64` is in fact below `40.63%` so IMO such check should fail. WDYT?

carmanchris31 commented:
That's a valid point. I can update to use floor.

krzkaczor commented:
@carmanchris31 would be great! Thanks!

carmanchris31 commented:
I reconsidered the approach after what we discussed and updated accordingly. Fewer changes now and I added a precision param and a couple comments for the future. Let me know what you think.

krzkaczor commented:
Looks great @carmanchris31. Thanks for all your work!

krzkaczor commented:
Released as `typecov@0.2.3`
